backend/entities/clubs/leadership/models.go

```diff
@@ -1,12 +1,12 @@
-package pocs
+package leadership
 
-type CreatePointOfContactBody struct {
+type CreateLeaderBody struct {
```

Comment:
nit but should this be leadership instead of leader? leader implies something different